README.md

```diff
@@ -6,3 +6,8 @@ at the moment. They might not be up to date.
 
 The branch names begin with a prefix indicating a package manager, followed by the
 test case name in the format: `<package-manager>/<test-case>`.
+
+To create a new test please create a branch in GitHub UI first and then open a PR
+against this branch. Alternatively start with pushing an empty branch first.
```

Comment:
s/Alternatively/Alternatively,/
s/start with pushing/you can push .... _if you have push rights._